src/tss2-esys/esys_tcti_default.c

```diff
@@ -40,6 +64,9 @@ struct {
 #ifdef _WIN32
     { .init = Tss2_Tcti_Tbs_Init, .conf = "",
       .description = "Access to TBS" },
+#elif defined (__VXWORKS__)
+    { .init = Tss2_Tcti_Device_Init, .conf = "/tpm0",
+      .description = "Access to /tpm0" },
```

Comment:
I assume your intention is to set the default for vxworks in terms of a "distribution"
You should use the `./configure --with-tctidefaultmodule --with-tctidefaultconfig" for this.

Reply:
The build configuration for VxWorks defines NO_DL. If added --with-tctidefaultmodule --with-tctidefaultconfig options, the build will fail.

Reply:
Ok, I looked at the code. What currently happens, if NO_DL is passed is
```
    { .init = Tss2_Tcti_Device_Init, .conf = "/dev/tpmrm0",
      .description = "Access to /dev/tpmrm0" },
    { .init = Tss2_Tcti_Device_Init, .conf = "/dev/tpm0",
.description = "Access to /dev/tpm0" },
```
This should work vor VxWorks, correct ?

Reply:
You mean to use /dev/tpmrm0 or /dev/tpm0? It could work but is not a perfect solution.
VxWorks doesn’t support resource manager. /dev/tpmrm0 is nonexistent. ESYS always try to open this device first. It will waste time to open a nonexistent and impact the performance.
VxWorks uses /tpm0 as the device name. If using /dev/tpm0, it needs to change the OS build configuration and impact the compatibility.